backend/notifications/push.ts

```
@@ -39,16 +39,19 @@ export async function sendPushNotification(title: string, body: string, users: U
+      store.all((err, sessions) => {
+        if (err) {
+          return reject(err)
+        }
+        if (Array.isArray(sessions)) {
+          resolve(sessions)
+        } else {
+          resolve([])
+        }
+      })
+    }
   })
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)